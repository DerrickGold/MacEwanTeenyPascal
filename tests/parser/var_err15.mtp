(* variable section start *)

(*missing array*)
var test: (20) of integer;

begin

end
.
