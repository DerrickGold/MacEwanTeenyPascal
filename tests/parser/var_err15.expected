Syntax Error: line 4:
Unexpected token (TOK_LPAREN, '('), expected a token type of:  
  'TOK_KEY_ARRAY' or
  'TOK_KEY_INTEGER'
